apps/reference/docs/guides/with-react.mdx

```diff
@@ -168,10 +168,10 @@ npx create-react-app supabase-react
 cd supabase-react
 ```
 
-Then let's install the only additional dependency: [supabase-js](https://github.com/supabase/supabase-js)
+Then let's install the only additional dependency: [supabase-js](https://github.com/supabase/supabase-js). We're going to install the "rc" (release candidate) version until this is released for general use.
```

Comment:
@silentworks  a little unsure about the language here, if you've got a better suggestion.